pkg/apis/integreatly/v1alpha1/rhmi_types.go

```diff
@@ -71,18 +71,19 @@ var (
 	PreflightSuccess    PreflightStatus = "successful"
 	PreflightFail       PreflightStatus = "failed"
 
+	// Operator image tags
 	OperatorVersionAMQStreams          OperatorVersion = "1.1.0"
-	OperatorVersionAMQOnline           OperatorVersion = "1.3.1"
+	OperatorVersionAMQOnline           OperatorVersion = "1.3"
 	OperatorVersionMonitoring          OperatorVersion = "1.1.4"
 	OperatorVersionSolutionExplorer    OperatorVersion = "0.0.45"
 	OperatorVersionRHSSO               OperatorVersion = "8.0.1"
 	OperatorVersionRHSSOUser           OperatorVersion = "8.0.1"
-	OperatorVersionCodeReadyWorkspaces OperatorVersion = "2.0.0"
-	OperatorVersionFuse                OperatorVersion = "7.5.0"
-	OperatorVersion3Scale              OperatorVersion = "0.4.0"
+	OperatorVersionCodeReadyWorkspaces OperatorVersion = "2.0"
+	OperatorVersionFuse                OperatorVersion = "1.5.0"
+	OperatorVersion3Scale              OperatorVersion = "1.10-4"
```

Comment:
Seems like the actual 3Scale operator version is `0.4.0` based on their version [here](https://github.com/3scale/3scale-operator/blob/v0.4.0/version/version.go#L4) which doesn't match their image tag. I would be inclined to change this back to `0.4.0` since that is what the actual operator version is within their operator.

wdyt @dimitraz @philbrookes? pinging @laurafitzgerald for visibility

Reply:
yes, let's use the actual operator version, rather than the image tag. Do we know why they are using 2 different versions like this?

Reply:
Sounds good, thanks guys. I'll get in touch with the 3scale team to ask about this

Reply:
Apparently there is no relation between the operator version and the operator image tag. I've updated the 3scale version now